ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/proxy/cco-manual-workload-identity/filestore-csi/provision/cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity-filestore-csi-provision-chain.yaml

```diff
@@ -0,0 +1,22 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity-filestore-csi-provision
+  steps:
+  - ref: gcp-provision-minimal-permission
+  - ref: gcp-provision-vpc
+  - ref: ignition-bastionhost
+  - ref: gcp-provision-bastionhost
+  - chain: ipi-conf-gcp
+  - ref: ipi-conf-proxy
+  - ref: ipi-conf-manual-creds
+  - ref: ipi-conf-gcp-oidc-creds-provision
+  - ref: ipi-conf-manual-creds-remove-unnecessary-creds
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-gcp-credentials-mode
```

Comment:
I think we can directly replace the above steps with `chain: cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity-provision`